met-web/src/components/engagement/view/widgets/PhasesWidget/PhasesWidgetMobile/EngagementPhaseMobile.tsx

```diff
@@ -12,32 +14,42 @@ export const EngagementPhaseMobile = ({
     learnMoreText,
     popOverText,
     accordionBackground,
+    phaseId,
 }: ProcessStageProps) => {
+    const { widgets } = useContext(ActionContext);
+    const phasesWidget = widgets.find((widget: Widget) => widget.widget_type_id === WidgetType.Phases);
+    const currentPhase = phasesWidget?.items[0]?.widget_data_id || EngagementPhases.Standalone;
+    const isCurrent = phaseId >= currentPhase;
+
     return (
         <PhaseBoxMobile
             title={title}
-            backgroundColor={backgroundColor}
+            backgroundColor={isCurrent ? backgroundColor : PAST_PHASE.backgroundColor}
             learnMoreBox={
-                <ReadMoreBox backgroundColor={accordionBackground} sx={{ p: 0, margin: 0 }}>
+                <ReadMoreBox
+                    backgroundColor={isCurrent ? accordionBackground : PAST_PHASE.backgroundColor}
+                    sx={{ p: 0, margin: 0 }}
+                >
                     <Grid
-                        sx={{ color: 'black' }}
+                        sx={{ color: isCurrent ? 'black' : 'white' }}
```

Comment:
Are your sure that the text color is black. Our primary text color is #494949. For this you can add the color: white conditionally. sx={ [ isCurrent && { color: 'white'} ]}

Reply:
In Marie Anik's mobile design the text did not look like our primary color. Ill change it to the default though.